src/index.scss

```diff
@@ -1,3 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Philosopher&display=swap');
+
+// Font Variables
+$inter: 'Inter', sans-serif;
+$philosopher: 'Philosopher', sans-serif;
```

Comment:
Are you using this font style somewhere?

Reply:
Yes, for headings both philosopher, inter and for body inter

Reply:
Yes philosopher is not being used. I was a bit confused on whether to set the headings to inter or philosopher